framework/src/org/checkerframework/common/value/qual/MinLenFieldInvariant.java

```diff
@@ -1,4 +1,4 @@
-package org.checkerframework.checker.index.qual;
+package org.checkerframework.common.value.qual;
```

Comment:
The Javadoc (below) is out-of-date.  Also, would it make more sense for this to be changed to ArrayLenRangeFieldInvariant?  (This is something I could do.)